pkg/config/config.go

```diff
@@ -84,21 +80,6 @@ func (n NodeConfig) HasDebugPod() bool {
 // DefaultTimeout for creating new interactive sessions (oc, ssh, tty)
 var DefaultTimeout = time.Duration(defaultTimeoutSeconds) * time.Second
 
-// Extract a container IP address for a particular device.  This is needed since container default network IP address
```

Comment:
this is no longer needed because we use whereabouts in our local-test-infra now?

Reply:
I am not sure why getting the IP from the container directly was needed as it should be reported through kubernetes API's status.podIPs (see below since PodIP v1 core). 
I will double check on a openshift 1.6 cluster, maybe it was an older api?


podIPs PodIP arraypatch strategy: mergepatch merge key: ip | podIPs  holds the IP addresses allocated to the pod. If this field is  specified, the 0th entry must match the podIP field. Pods may be  allocated at most 1 value for each of IPv4 and IPv6. This list is empty  if no IPs have been allocated yet.
-- | --

Reply:
Ok, it's fine if you think it's not needed in general. Just making sure we are not coding the test specific to our local-test-infra.

Reply:
verified that the object is present in ocp 4.6